cli/resolver.rs

```diff
@@ -66,8 +67,7 @@ pub struct ModuleCodeStringSource {
 
 #[derive(Debug)]
 pub struct CliNodeResolver {
-  // not used in the LSP
-  cjs_resolutions: Option<Arc<CjsResolutionStore>>,
+  cjs_resolutions: Arc<CjsResolutionStore>,
```

Comment:
This means we have this cache in two places for the LSP, which is not ideal.

Reply:
I guess we can address this in a follow up?